third_party/subscriptions-project/swg.js

```diff
@@ -6643,7 +6643,7 @@ class LinkSaveFlow {
         });
       }
     });
-    /** {!Promise<boolean>} */
+    /** @type {!Promise<boolean>} */
     return this.dialogManager_.openView(this.activityIframeView_,
```

Comment:
are you casting here or just declaring type?

Reply:
ah good point. this was a blind change and I didn't really look into it until now.  Not sure why this is here when its not doing any casting..I'll just remove.